src/components/Calculator.js

```diff
@@ -1,74 +1,111 @@
 import '../calculator.css';
+import Big from 'big.js';
+import React, { useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉